_posts/2017-06-20-meetup.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Meep! AustinJS is BACK! Meep Meep!
+author: lon
+layout: post
+categories:
+- posts
+- meetups
+when: 2017-06-20T19:30:00-05:00
+---
+
+<img alt="Beaker" src="https://upload.wikimedia.org/wikipedia/en/5/59/Beaker_%28Muppet%29.jpg" />
+
+Hey friends! AustinJS is back with an exciting dual talk from Austin's own [Tara Vancil](https://twitter.com/taravancil) and [Paul Frazee](https://twitter.com/pfrazee) about their project, [Beaker Browser][Beaker], an experimental web browser with builtin hosting and p3p web protocols. 
+
+<div class="media-object speaker-bio">
+  <a href="https://twitter.com/taravancil">
+    <img alt="Tara Vancil @taravancil on Twitter"
+      src="https://pbs.twimg.com/profile_images/844315037351096321/qLBLZqkl_400x400.jpg" />
+  </a>
+  <div>
+  <a href="https://twitter.com/taravancil"><strong>Tara Vancil</strong></a>
+  helps build [Beaker][], a browser for the peer-to-peer Web. She’s enthusiastic
+  about decentralizing the Web, and thinks that peer-to-peer protocols will
+  reinvigorate the creativity of the Web’s early days.
+  </div>
+</div>
+
+<div class="media-object speaker-bio">
+  <a href="https://twitter.com/pfrazee">
+    <img alt="Paul Frazee @pfrazee on Twitter"
+      src="https://pbs.twimg.com/profile_images/868195193253535744/yXQt-VYo_400x400.jpg" />
+  </a>
+  <div>
+  <a href="https://twitter.com/pfrazee"><strong>Paul Frazee</strong></a>
+  works on a peer-to-peer browser called [Beaker][]. He lives in Austin TX.
```

Comment:
Small change: "Paul Frazee works on Beaker, and is also very enthusiastic about a decentralized Web. He lives in Austin TX."